src/data/events.json

```diff
@@ -174,5 +174,17 @@
     "location": "Chennai",
     "communityName": "TechNexus Community",
     "communityLogo": "https://secure.meetupstatic.com/photos/event/5/2/7/5/clean_525381109.webp"
-  }
+  },
+  {
+  "eventName": "Swift Chennai Chapter #01",
+  "eventDescription": "Swift Chennai ​Chapter #01 marks the start of a fresh, vibrant meetup series for Swift devs, iOS enthusiasts, and curious creators. Come for the code, stay for the kaapi, and leave with solid connections.",
+  "eventDate": "2025-04-26", // Format: YYYY-MM-DD
```

Comment:
Hey @rizwan95 
Thanks a lot for your contribution. 
Love to see Swift Chennai come up.

There are few things that would be needed to change to get this into prod, once we are done with that. I'll merge it ASAP. 

Thanks a lot

Cheers :)